weave

```diff
@@ -801,6 +857,7 @@ case "$COMMAND" in
     expose)
         collect_cidr_args "$@"
         shift $CIDR_COUNT
+        [ $CIDR_COUNT -gt 0 ] || { get_ipam_cidr weave && expose $CIDR_ARGS ; }
```

Comment:
I think at the time this was written I was unclear how to get hold of the specific IP address to hide; however this isn't necessary since we can ask IPAM to release _all_ addresses (effectively only one) for the container name ("weave").

the UX will get more complicated when IPAM supports multiple subnets.